theroute/src/styles/ViewExpenses.css

```diff
@@ -148,8 +149,8 @@ h1 {
 

```

Comment:
Try to add comments about what the css styling does, for example why 90 instead of 70%

Reply:
okay, just added some comments about the styling